source/includes/limitations-filtering.rst

```diff
@@ -0,0 +1,20 @@
+- Filtering is not supported with :ref:`reversible sync
+  <c2c-api-reverse>`.
+- The destination cluster must not contain user data prior to starting.
+- The destination cluster must not contain the
+  ``mongosync_reserved_for_internal_use`` system database prior to
+  starting.
+- You cannot modify a filter that is in use. To create a new filter, see
+  :ref:`changing the filter <c2c-change-filter>`.
+- If you rename a collection or database that is part of a filter,
```

Comment:
This is confusing. It could continue syncing, if the new namespace is in the filter. There are special cases for within DB and cross DB renames.